.travis.yml

```diff
@@ -31,6 +31,16 @@ script: coverage run --source=trollsched setup.py test
 after_success:
 - if [[ $PYTHON_VERSION == 3.6 ]]; then coveralls; fi
 #
+deploy:
+  - provider: pypi
+    user: adybbroe
+    password:
+      secure: SY0qo7sZXDjDx0DHvuXrHvL9VTAulgU/T33d6UWXf469jT9DOexuZ2VYLgJbYQen5FSe5JmQE0ZMdId1cb8IPP/77qCgQK6f0lRDa43fSYXhcD+fHzlQskievJrwamkRYx6WBrJbwGAKBNinUgNSaTdbh9XUugziGFiOHUfVppM=
+    distributions: sdist bdist_wheel
```

Comment:
I think you'll want to remove the wheel. I don't think it provides any benefit. If you really want the wheel I think you should look at adding the information to setup.cfg to say `universal=1` in a `bdist_wheel` section. That is, if this is a pure python package.

Reply:
Thanks, done